V1.0.0

cat-haines opened the pull request:
Mostly just updating the documentation.

cat-haines committed:
v1.0.0 - mostly documentation changes

cat-haines committed:
Tiny README changes

ersatzavian commented:
Some comments: 
- Why is there a console.log in the README?
- Not sure I like the use of the degree symbol instead of the word degrees throughout the comments. Just because we can do that doesn't mean we should. 
- You've changed _getShutdown to getShutdown and _getExtMode to getExtMode, why? These aren't intended as public methods, even if they are proper get methods. 
- One-line if statements are within the style guide, as long as they're curly-braced. I don't like making these into three lines if we don't have to:

```
if (something) { do one line of things; }
else { do something else; }
```

Reads cleaner to me than 

```
if (something) {
    do something;
} 
else {
    do something else;
}
```

All of the various code standards I've used before also require "cuddled" else statements, which I vastly prefer to starting a line with "else". There may be other reasons why to or not to do this, but to my eye it keeps the else visually grouped with its if/else if family:

```
if (something) {
    do something;
} else {
    do something else;
}
```

cat-haines committed:
Changed ºC to degrees C
Rewrote some if statements to make in line
Changed a console.log to server.log in the README

cat-haines commented:
Fixed the issues you commented on.
- console.log was a typo.. fixed :)
- º -> degrees
- I changed the if statements to inline if statements (with the exception of if statements that are returning things).
- I made getExtMode and getShutdownMode public since we had setters, but not getters.. seems rude to let people write a bit, but not read that bit :)

You can merge if/when you're happy!

ersatzavian commented:
All good, thanks!
